### PR TITLE
docs: minor improvements

### PR DESCRIPTION
**Pull request by AyanavaKarmakar**
## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

- TailwindCSS => Tailwind CSS (Missed one in https://github.com/t3-oss/create-t3-app/pull/748)
- Fix typos, punctuation, and grammar

---

💯

### CODE REVIEW

File: www/src/pages/en/usage/prisma.md [hunk 1]
```diff
@@ -16,11 +16,11 @@ You will find the Prisma schema file at `/prisma/schema.prisma`. This file is wh
...
+When you select NextAuth.js in combination with Prisma, the schema file is generated and set up for you with the recommended values for the `User`, `Session`, `Account`, and `VerificationToken` models, as per the [NextAuth.js documentation](https://next-auth.js.org/adapters/prisma).
 
 ## Default Database
 
-The default database is a SQLite database, which is great for development and quickly spinning up a proof-of-concept, but not recommended for production. You can change the database to use by changing the `provider` in the `datasource` block to either `postgresql` or `mysql`, and then updating the connection string within environment variables to point to your database.
+The default database is an SQLite database, which is great for development and quickly spinning up a proof-of-concept but is not recommended for production. You can change the database to use by changing the `provider` in the `datasource` block to either `postgresql` or `mysql`, and then updating the connection string within environment variables to point to your database.
```
Comment (nexxeln) [inline www/src/pages/en/usage/prisma.md:23]: a SQLite database is correct no?

Comment (AyanavaKarmakar) [reply www/src/pages/en/usage/prisma.md:23]: image

Refer: https://www.grammar.com/a-vs-an-when-to-use

Comment (juliusmarminge) [reply www/src/pages/en/usage/prisma.md:23]: It you don't say S Q Lite do you? Or is that just me

Comment (AyanavaKarmakar) [reply www/src/pages/en/usage/prisma.md:23]: Is there a "correct" pronunciation? Been saying S Q Lite all this time

Comment (AyanavaKarmakar) [reply www/src/pages/en/usage/prisma.md:23]: image

Refer: https://database.guide/is-it-pronounced-s-q-l-or-sequel/

**Comment by AyanavaKarmakar** [comment]
Looks like some of the index pages have the wrong links

**Review by juliusmarminge — APPROVED** [review]

**Comment by juliusmarminge** [comment]
We'll need changesets for the links on the index'es to go live

**Pull request merged**